Cargo.toml

```diff
@@ -14,13 +14,16 @@ license = "MIT OR Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/ProbablyClem/utoipauto"
 homepage = "https://github.com/ProbablyClem/utoipauto"
+
 [lib]
 
+[features]
+generic_full_path = ["utoipauto-macro/generic_full_path"]
 
 [dependencies]
-utoipauto-macro = "0.1.9"
+utoipauto-macro = { path = "utoipauto-macro", version = "0.1.9"}
```

Comment:
Unfortunatly, we can't merge this.
Crates needs to be published one by one to crates.io and cannot depend on each other locally.
It's an annoying issue but I couldn't find a way to fix it.

I change locally when I work on the project but I put it back when I push...

Reply:
Apparently it should work https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies
Let's try, I'll fix it if it's still an issue

Reply:
Okay :+1: